### ISSUE TITLE
relative path from /foo/qux and /foobar/qux is ./bar/qux

### ISSUE THREAD

**Issue by wvanderdeijl** [smysnk/gulp-rev-all#134]
when the name of a directory is the name of a sibling directory with some additional characters the relative paths between these two directories are calculated wrong.

For example when getting a relative path from `/foo/index.html` to `/foobar/index.html` this returns `./bar/index.html` and not `../foobar/index.html`

example of failing tests:
```
    it('relative path bug', function () {
        // relative path
        var file = new Gutil.File({
            path: '/first/second/third/index.html',
            base: '/first/second'
        });
        var fileReference = new Gutil.File({
            path: '/first/second/thirder/index.html',
            base: '/first/second'
        });
        file.revPathOriginal = file.path;
        fileReference.revPathOriginal = fileReference.path;
        Tool.get_reference_representations_relative(fileReference, file).should.eql(['../thirder/index.html']);
    });
    it('absolute path bug', function () {
        // absolute path
        file = new Gutil.File({
            path: '/first/second/thirder/index.html',
            base: '/first/second'
        });
        fileReference = new Gutil.File({
            path: '/first/second/third/index.html',
            base: '/first/second'
        });
        file.revPathOriginal = file.path;
        fileReference.revPathOriginal = fileReference.path;
        Tool.get_reference_representations_absolute(fileReference, file).should.eql(['/third/index.html','third/index.html']);
    });
```

**Comment by wvanderdeijl** [comment]
This caused a reference in `/foo/test.js` to `/foobar/other.js` to not be replaced as it is trying to find a reference `./bar/other.js` or `bar/other.js`

**Comment by smysnk** [comment]
Resolved with #135
